contracts/utils/contracts/test/TestLogDecodingDownstream.sol

```diff
@@ -1,37 +0,0 @@
-/*
```

Comment:
I don't see the file that this tests in the diff. Is this test still in use?

Reply:
Same as above